framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2203,38 +2177,59 @@ private boolean isAnnotatedForThisCheckerOrUpstreamChecker(@Nullable Element elt
     }
 
     /**
-     * Determine the standard set of suppress warning keys usable for any checker.
+     * Returns a modifiable set of lower-case strings that are prefixes for SuppressWarnings
+     * strings.
      *
-     * @see #getSuppressWarningsKeys()
-     * @return collection of warning keys
+     * <p>The collection must not be empty and must not contain only {@link #SUPPRESS_ALL_PREFIX}.
+     *
+     * @return non-empty modifiable set of lower-case prefixes for SuppressWarnings strings
+     */
+    public SortedSet<String> getSuppressWarningsPrefixes() {
+        return getStandardSuppressWarningsPrefixes();
+    }
+
+    /**
+     * Returns a sorted set of SuppressWarnings prefixes read from the {@link
+     * SuppressWarningsPrefix} meta-annotation on the checker class. Or if no {@link
+     * SuppressWarningsPrefix} is used, the checker name is used. {@link #SUPPRESS_ALL_PREFIX} is
+     * also added, at the end.
+     *
+     * @return a sorted set of SuppressWarnings prefixes
      */
-    protected final Collection<String> getStandardSuppressWarningsKeys() {
-        // TreeSet ensures keys are returned in a consistent order.
-        Set<String> result = new TreeSet<>();
-        result.add(SUPPRESS_ALL_KEY);
+    protected final NavigableSet<String> getStandardSuppressWarningsPrefixes() {
+        NavigableSet<String> prefixes = new TreeSet<>();
+        prefixes.add(SUPPRESS_ALL_PREFIX);
```

Comment:
I wonder if "allcheckers" should be put at the end of the list rather than the beginning, so it is less prominent in the output of `-AshowSuppressWarningStrings`.

Reply:
It's a TreeSet, so it is ordered by name.